Makefile

```diff
@@ -11,7 +11,7 @@ MKFILE_DIR := $(dir $(MKFILE_PATH))
 RELEASE_DIR := ${MKFILE_DIR}bin
 
 # Version
-RELEASE?=1.0.1
+RELEASE?=1.1.0
```

Comment:
Make it consistent: v1.1.0

Reply:
Updated

Reply:
the docker tag becomes v1.1.0, is it right?

Reply:
In https://hub.docker.com/repository/docker/megaease/easegress, its tags are `v1.0.1`, `v1.0.0`.

Reply:
Yes, please remember update tag `server-sidecar`.

Reply:
I checked several other applications, they don't have a `v` in the version tag.

Reply:
> Yes, please remember update tag `server-sidecar`.

Sure. I will update it after we finished this releasing process.

Reply:
https://semver.org/#is-v123-a-semantic-version
https://github.com/etcd-io/etcd/releases

Reply:
> I checked several other applications, they don't have a `v` in the version tag.

Emm. But we had already launched `v1.0.0`, and `v1.0.1`. If we chose to trim this `v` from the version tag, will it become inconsistent?